column.go

```diff
@@ -199,18 +202,18 @@ func (c *Column) forEachLeaf(do func(*Column)) {
 	}
 }
 
-func openColumns(file *File) (*Column, error) {
+func openColumns(file *File, metadata format.FileMetaData, columnIndexes []format.ColumnIndex, offsetIndexes []format.OffsetIndex) (*Column, error) {
```

Comment:
It seems that we are typically passing FileMetaData by pointer in other places, let's do it that way here for consistency as well?